website/source/docs/providers/aws/r/api_gateway_integration.html.markdown

```diff
@@ -56,9 +56,10 @@ The following arguments are supported:
   Not all methods are compatible with all `AWS` integrations.
   e.g. Lambda function [can only be invoked](https://github.com/awslabs/aws-apigateway-importer/issues/9#issuecomment-129651005) via `POST`.
 * `request_templates` - (Optional) A map of the integration's request templates.
+* `request_parameters` - (Optional) Request query string parameters and headers that should be passed to the
 * `passthrough_behavior` - (Optional) The integration passthrough behavior (`WHEN_NO_MATCH`, `WHEN_NO_TEMPLATES`, `NEVER`).  **Required** if `request_templates` is used.
 * `request_parameters_in_json` - (Optional) A map written as a JSON string specifying
```

Comment:
a nitpick (i.e. something I can fix after merge), but it would help if you marked this field as deprecated here. - e.g. changed the whole line to
```
* `request_parameters_in_json` - **Deprecated**, use `request_parameters` instead.
```